qt/python/mantidqt/widgets/fitpropertybrowser/fitpropertybrowser.py

```diff
@@ -36,14 +35,13 @@ class FitPropertyBrowser(FitPropertyBrowserBase):
     """
 
     closing = Signal()
-    pattern_fittable_curve = re.compile('(.+?): spec (\d+)')
 
-    def __init__(self, canvas, toolbar_state_checker, parent=None):
+    def __init__(self, canvas, toolbar_state_checker, fig_manager, parent=None):
```

Comment:
The `ToolbarStateChecker` was created to check the toolbar without going through the `fig_manager`. Can we use that instead of requiring the `fig_manager` too?